pibooth/booth.py

```diff
@@ -33,7 +33,8 @@
 except ImportError:
     from gpiozero.pins.mock import MockFactory
     Device.pin_factory = MockFactory()
-    LOGGER.debug("Programme not start on Raspberry pi use MockFactory() mode")
+    LOGGER.debug("NMockFactory() mode")
```

Comment:
Thinks there is a typo here: what is NMockFactory?

Reply:
corrected